web_src/js/components/ContextPopup.vue

```diff
@@ -91,16 +91,16 @@ export default {
 <template>
   <div ref="root">
     <div v-if="loading" class="tw-h-12 tw-w-12 is-loading"/>
-    <div v-if="!loading && issue !== null">
-      <p><small>{{ issue.repository.full_name }} on {{ createdAt }}</small></p>
-      <p><svg-icon :name="icon" :class="['text', color]"/> <strong>{{ issue.title }}</strong> #{{ issue.number }}</p>
-      <p>{{ body }}</p>
+    <div v-if="!loading && issue !== null" class="tw-flex tw-flex-col tw-gap-2">
+      <div class="tw-text-12">{{ issue.repository.full_name }} on {{ createdAt }}</div>
+      <div><svg-icon :name="icon" :class="['text', color]"/> <strong>{{ issue.title }}</strong> #{{ issue.number }}</div>
```

Comment:
It needs `flex-text-block`, otherwise svg doesn't align with text

Reply:
Adding `flex-text-block` fixes the alignment but would introduce a overflow issue:

<img width="1238" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/030ef6d7-2728-4f2f-8507-18be15423ccf">

Reply:
That's what we have discussed about how to limit the text in a flex box ........

Reply:
But I am fine to keep it as-is, without the flexbox

Reply:
Fixed:

<img width="616" alt="Screenshot 2024-05-02 at 14 21 24" src="https://github.com/go-gitea/gitea/assets/115237/65b040ab-eeda-47f1-9b9f-04b874e61dda">